### PR TITLE
[codestyle] Use isEmpty() instead if .size()

### PR DESCRIPTION
**Pull request by SrijanJha28**
1. http://sonar.xwiki.org/issues/search#issues=a434e23e-2cd5-4b77-b43b-2e96d6f7c186
2. http://sonar.xwiki.org/issues/search#issues=a61ea655-74ff-4a8e-b035-5e6532fe1777
3. http://sonar.xwiki.org/issues/search#issues=5eb04857-681d-4136-a521-bdff4bc820c6
4. http://sonar.xwiki.org/issues/search#issues=cbff6b56-b63b-4476-8c79-ff124d4fa4b8
5. http://sonar.xwiki.org/issues/search#issues=8fb93de5-42e2-4306-b66e-e178aa8323fa

### CODE REVIEW

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java [hunk 1]
```diff
@@ -236,15 +236,15 @@ public void printVerbatimContent(String verbatimContent)
             }
         }
 
-        if (subVerbatimStack.size() > 0) {
+        if (!subVerbatimStack.isEmpty()) {
```
Comment (vmassol) [inline xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java]: Be careful you change the logic! :(

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java [hunk 2]
```diff
@@ -236,15 +236,15 @@ public void printVerbatimContent(String verbatimContent)
...
             // Append remaining string
             subVerbatimStack.peek().append(end);
 
             // Escape not closed verbatim blocks
-            while (subVerbatimStack.size() > 0) {
+            while (!subVerbatimStack.isEmpty()) {
```
Comment (vmassol) [inline xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java]: same here

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java (hunk 1 above)
Comment (SrijanJha28) [reply xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java]: But it is correct , when the size is not empty it should be greater than 0 (as it cannot be negative)

Comment (vmassol) [reply xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java]: Indeed you're right sorry, went too fast in my review :)

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java (hunk 2 above)
Comment (vmassol) [reply xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java]: TBH I'm personally finding it easier to read "subVerbatimStack.size() > 0" than "!subVerbatimStack.isEmpty()" .... :) But let's consider it's not a false positive for now...

**Pull request merged**